db-scripts/src/main/resources/migration.sql

```diff
@@ -380,4 +380,49 @@ ALTER TABLE `mutation` ADD COLUMN `DRIVER_FILTER_ANNOTATION` VARCHAR(80) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER` VARCHAR(50) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER_ANNOTATION` VARCHAR(80) NULL;
 
-UPDATE info SET DB_SCHEMA_VERSION="2.4.0";
+##version: 2.4.1
+-- ========================== new reference genome genes related tables =============================================
+CREATE TABLE `reference_genome` (
+    `REFERENCE_GENOME_ID` int(4) NOT NULL AUTO_INCREMENT,
+    `SPECIES` varchar(64) NOT NULL,
+    `NAME` varchar(64) NOT NULL,
```

Comment:
this `NAME` is the "hg" name and the `BUILD_NAME` is the "GRCh" name? Why is `NAME` not unique?

Reply:
I guess is the historical reason to have a build name for each reference genome to distinguish different genome builds form different publishers, for example, NCI build from UCSC build. the NCI genome build normally has a prefix of "GRCh". 'hg' is stands for human genome. It is better to keep the way.

Reply:
please also add 
```
UNIQUE INDEX `NAME_UNIQUE` (`NAME` ASC)
```